src/NServiceBus.Core/Sagas/SagaInvocationResult.cs

```diff
@@ -1,16 +1,21 @@
 namespace NServiceBus
 {
+    using Sagas;
+
     class SagaInvocationResult
     {
+        public SagaMetadata SagaMetadata { get; private set; }
```

Comment:
I was thinking about this property. What if the saga was found and someone wants to access this property? It will be null. We could rather make this private and rename it to be more specific like NotFoundSagaTypeName. I am also proposing [removing the type reference altogether.](https://github.com/Particular/NServiceBus/pull/6248/files#r798515491)

Reply:
Look at my other comments, and I think we can delete this property altogether.